### DISCUSSION TITLE
Simple Products not showing options when grouped

### DISCUSSION THREAD

**Discussion by ghost** [OpenMage/magento-lts#2085] [Ideas]
This is more like an improvement than a bug. I think this is actually critical to add; I know some extensions do this, but most break up other modules at least for me it did.

**Comment by seansan** [comment]
I suggest to close this issue. @Flyingmana 
The error is not clear. And the user has gone away.

**Comment by Flyingmana** [comment]
interesting, thats a new for me.

Regarding the Issue, its a Feature Request, not an error.
And showing product options in grouped articles may be of interest for someone.

**Comment by fballiano** [comment]
this thread has some interesting links:
https://community.magento.com/t5/Magento-2-x-Feature-Requests-and/Feature-Suggestion-Configurable-products-in-Bundle-and-or-Groups/idi-p/46200

I use configurablebundle by wizkunde for a customer of mine, we should contact them and ask to open source the extension, since their website seems to be down.
